cmd/kusk/.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @jasmingacic @kylehodgetts @povilasv
```

Comment:
Is this a valid place to put the file? I thought it should live in the top-level part of the repository.